jax/_src/numpy/lax_numpy.py

```diff
@@ -3341,11 +3341,40 @@ def copy(a: ArrayLike, order: str | None = None) -> Array:
   return array(a, copy=True, order=order)
 
 
-@util.implements(np.zeros_like)
 def zeros_like(a: ArrayLike | DuckTypedArray,
                dtype: DTypeLike | None = None,
                shape: Any = None, *,
                device: xc.Device | Sharding | None = None) -> Array:
+  """Create an array full of zeros with the same shape and dtype as an array.
+
+  JAX implementation of :func:`numpy.zeros_like`.
+
+  Args:
+    a: Array-like object with ``shape`` and ``dtype`` attributes.
+    shape: optionally override the shape of the created array.
+    dtype: optionally override the dtype of the created array.
+    device: (optional) :class:`~jax.Device` or :class:`~jax.sharding.Sharding`
+      to which the created array will be committed.
+
+  Returns:
+    array of the specified shape and dtype, on the specified device if specified.
```

Comment:
Is it intentional to sometimes write "Array" (like in "a: Array-like ...") and other times "array" (like here)? Maybe we should always capitalize?
```suggestion
    Array of the specified shape and dtype, on the specified device if specified.
```

Reply:
Done, thanks!